test/avocado/selenium-base.py

```diff
@@ -7,27 +7,31 @@
 if not machine_test_dir in sys.path:
     sys.path.insert(1, machine_test_dir)
 
-from avocado import main
-from avocado.utils import process
-from seleniumlib import SeleniumTest, user, clickable, passwd
+from seleniumlib import SeleniumTest, user, clickable, passwd, visible
 
 class BasicTestSuite(SeleniumTest):
     """
     :avocado: enable
     """
     def test10Base(self):
-        out = process.run("hostname", shell=True)
+        self.wait_id('server-name')
+        self.error = False
```

Comment:
This test now does practically nothing. I think it'd be better to move the wait_id() into the tests that need it.